ecocodepython.properties

```diff
@@ -24,3 +24,9 @@ defaults.mavenArtifactId=ecocode-python-plugin
 1.4.2.date=2024-01-11
 1.4.2.downloadUrl=https://github.com/green-code-initiative/ecoCode-python/releases/download/1.4.2/ecocode-python-plugin-1.4.2.jar
 1.4.2.changelogUrl=https://github.com/green-code-initiative/ecoCode-python/releases/tag/1.4.2
+
+1.4.3.description=Add support for SonarQube 10.4 "DownloadOnlyWhenRequired" feature; correction setter problem on constructor method; check Sonarqube 10.4.1 compatibility
+1.4.3.sqVersions=[9.9,10.4.1]
```

Comment:
See my comment on the Java plugin
AND did you really mean to end this at 10.4.1 rather than 10.5.1?

Reply:
Hi @ganncamp , please see my comment on java plugin ...
we have an issue on python plugin with 10.5.x Sonarqube versions.
thus, 10.4.1 really wanted.